src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -34,8 +36,15 @@ public FranchiseeAndStaffDto signUpFranchiseeAndStaff(@RequestBody @Valid Franch
 
     @PostMapping("/{franchiseeId}/service_areas")
     @ResponseStatus(HttpStatus.CREATED)
-    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto, @PathVariable(value = "franchiseeId") Long franchiseeId) {
+    public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto suburbListPostDto,
+                                         @PathVariable(value = "franchiseeId") Long franchiseeId) {
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @PostMapping("/{franchiseeId}/accept-orders")
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
same like you just copied, so why returns CREATED?

Reply:
The backend of service areas was also written when we cooperated before, and I didn't realize that no new resources were created, so I should use HttpStatus.OK